### PR TITLE
EVG-8229: query for aborted task info

### PR DESCRIPTION
**Pull request by tgrander**
https://jira.mongodb.org/browse/EVG-8229

When a task is aborted on the commit queue, we want to show a message on the task details page with information about the task that was aborted and its build. This query gets the abort info for a task that was aborted.

### CODE REVIEW

File: graphql/resolvers.go [hunk 1]
```diff
@@ -124,6 +124,34 @@ func (r *queryResolver) MyPublicKeys(ctx context.Context) ([]*restModel.APIPubKe
...
+
+	abortedTask, err := task.FindOne(task.ById(at.AbortInfo.TaskID))
+	if err != nil {
+		return &info, InternalServerError.Send(ctx, fmt.Sprintf("Problem getting aborted task %s: %s", *at.Id, err.Error()))
+	}
+	abortedTaskBuild, err := build.FindOne(build.ById(abortedTask.BuildId))
+	if err != nil {
+		return &info, InternalServerError.Send(ctx, fmt.Sprintf("Problem getting build for aborted task %s: %s", abortedTask.BuildId, err.Error()))
+	}
+	if abortedTaskBuild == nil || abortedTask == nil {
```
Comment (ablack12) [inline graphql/resolvers.go:145]: you should check abortedTask == nil before accessing buildID at 141

File: graphql/resolvers.go [hunk 2]
```diff
@@ -124,6 +124,34 @@ func (r *queryResolver) MyPublicKeys(ctx context.Context) ([]*restModel.APIPubKe
...
+	if at.Aborted != true {
+		return nil, nil
+	}
+
+	info := AbortInfo{
+		User:   &at.AbortInfo.User,
+		TaskID: &at.AbortInfo.TaskID,
+	}
+
+	abortedTask, err := task.FindOne(task.ById(at.AbortInfo.TaskID))
```
Comment (khelif96) [inline graphql/resolvers.go:137]: You can simplify this to `task.FindOneId()` https://github.com/evergreen-ci/evergreen/blob/05d7b78d23cd59007ec5378783ab5aa65fbb704b/model/task/db.go#L814

File: graphql/resolvers.go [hunk 3]
```diff
@@ -124,6 +124,34 @@ func (r *queryResolver) MyPublicKeys(ctx context.Context) ([]*restModel.APIPubKe
...
+	if err != nil {
+		return &info, InternalServerError.Send(ctx, fmt.Sprintf("Problem getting aborted task %s: %s", *at.Id, err.Error()))
+	}
+	abortedTaskBuild, err := build.FindOne(build.ById(abortedTask.BuildId))
```
Comment (khelif96) [inline graphql/resolvers.go:141]: Same comment as above build.FindOneId() 
https://github.com/evergreen-ci/evergreen/blob/05d7b78d23cd59007ec5378783ab5aa65fbb704b/model/build/db.go#L236

**Review by khelif96 — COMMENTED** [review]
Apart from Annies comment LGTM

**Review by SupaJoon — APPROVED** [review]

**Review by khelif96 — APPROVED** [review]

**Pull request merged**